### ISSUE TITLE
Create browsable web page

### ISSUE THREAD

**Issue by wilzbach** [wilzbach/d-mans#1]
Automatically generated Github Pages or another visualization format
for convenient browsing.

Ideas are welcome ;-)

**Comment by bytorbrynden** [comment]
~~I was planning on throwing something together quickly, however, unless you have another form of hosting, this might be quite difficult.~~

~~The reason why this might be difficult, is because GitHub pages requires that you put the page that you want to be "published" under the `gh-pages` branch.~~

~~My _plan_, was to create a(n) über simple set of pages (one or two, three at the very most), that would use relative paths to "fetch" the images.~~

Actually, I **can** throw something together, using the GitHub API and a simple `gh-pages` branch.

**Comment by wilzbach** [comment]
> I was planning on throwing something together quickly, however, unless you have another form of hosting, this might be quite difficult.

A couple of my ideas:
- Jekyll and Github pages (we probably have to manually list the images in the main yaml)
- my VPS (hosting is for free, deployment could be done with rsync, but no stability guarantees)
- AWS S3

> Actually, I can throw something together, using the GitHub API and a simple gh-pages branch.

Hey let's not make our lifes too complicated, I just saw that Jekyll supports iteration of static files builtin without the need of any plugins :)

``` ruby
{% for image in site.static_files %}
    {% if image.path contains 'images' %}
        <img src="{{ site.baseurl }}{{ image.path }}" alt="image" />
    {% endif %}
{% endfor %}
```

http://stackoverflow.com/questions/17677094/jekyll-for-loop-over-all-images-in-a-folder

**Comment by wilzbach** [comment]
btw it's great to hear from other D-man enthusiasts :)

**Comment by bytorbrynden** [comment]
I like your ideas. I don't have any experience with Jekyll, but I'd imagine it's probably not very difficult.

> Hey let's not make our lifes too complicated [ ... ]

The GitHub API is actually pretty straight-forward.

**Comment by wilzbach** [comment]
>  I don't have any experience with Jekyll, but I'd imagine it's probably not very difficult.

Yes, I quickly put [something](https://github.com/wilzbach/d-mans/blob/gh-pages/meiz/index.html) [together](https://wilzbach.github.io/d-mans/meiz/). Feel free to improve upon that (I am not a designer by any means).

**Comment by bytorbrynden** [comment]
That's pretty neat, actually. I don't think it needs any improvement.

I'm a terrible designed, so I can't help you there, sorry.
